CHANGES.rst

```diff
@@ -20,6 +20,11 @@ Fixes
 - Don't error out when showing permissions for a non-existent user
   (`#437 <https://github.com/zopefoundation/Zope/issues/437>`_)
 
+Other changes
```

Comment:
I think this is a breaking change as we remove a feature.

Reply:
Re-characterized as "Breaking change" in the change log now

Reply:
And changed WHATSNEW.rst to reflect the removal.